front-end/src/ui-components/Card.tsx

```diff
@@ -19,10 +19,10 @@ const Card = ({ className, children }:Props ) => {
 export default styled(Card)`
 	background-color: white;
 	padding: 2rem 3rem 2rem 3rem;
-	border-style: solid;
-	border-width: 1px;
-	border-color: grey_border;
 	border-radius: 3px;
+	-webkit-box-shadow: 5px 5px 5px 0px rgba(227,227,227,1);
+	-moz-box-shadow: 5px 5px 5px 0px rgba(227,227,227,1);
+	box-shadow: 5px 5px 5px 0px rgba(227,227,227,1);
```

Comment:
we have common values for the box shadow color, it's `box_shadow_secondary_grey`, would that work? (you need to set the box shadow color on a dedicated line).
Also we don't use any browser specific css so far, afaik, let's remove the `-webkit..` and `-moz..`

Reply:
Done